R/write_tests_xlsx.R

```diff
@@ -4,29 +4,30 @@
 #' @param name of dataset tests
 #'
 #' @return a workbook containing comparison tests with each dataset with a new tab
```

Comment:
This function doesn't return the workbook. It writes the workbook to disk. I'm not actually sure what it returns i.e. if you did `object <- write_tests_xlsx(...)` what would `object` be?

Some options are to return nothing `return(NULL)`, return TRUE to show that it worked. If this would be used in a pipe it might make more sense to either return the data `return(comparison_data)` (this is usually what the readr and other type functions do) or to return the file path where the xlsx was saved.

Reply:
This makes sense/I agree, the function writes the workbook to disk so i dont think we would use this applied to an object. We would use this in a pipe, so i think return the file path where the xlsx was saved makes sense to me. I've made this change and think this is ready now :)